config/getter.go

```diff
@@ -81,6 +81,15 @@ func GetProfilerPath() string {
 
 // -- server --
 
+func GetPort() *int {
+	if !viper.IsSet("port") {
```

Comment:
Can’t we set a default port in code and return an int instead of a pointer?

Reply:
yeah but it will be hard for us to tell if $PORT env is provided to us

Reply:
Ah word, that’s a good point.